### PR TITLE
FLOC-3913 Remove curls for missing files and postgres dir

### PR DESCRIPTION
**Pull request by sarum90**
fixes https://clusterhq.atlassian.net/browse/FLOC-3913

### CODE REVIEW

File: admin/installer/setup_client.sh [hunk 1]
```diff
@@ -15,12 +15,6 @@ sudo apt-get install -y postgresql-client
 curl -L https://github.com/docker/compose/releases/download/1.5.2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
 chmod +x /usr/local/bin/docker-compose
 
-mkdir -p /home/ubuntu/postgres
```
Comment (sarum90) [inline admin/installer/setup_client.sh]: For design review: Should we remove lines 10 and 11 as well?

Comment (sarum90) [inline admin/installer/setup_client.sh]: Chatted with Richard offline. Will wait for follow up to remove those depending on demo instructions.

**Comment by sarum90** [comment]
As discussed in person, request code review to involve testing https://s3-us-west-1.amazonaws.com/mewert-chq-work-bucket/FLOC-3913-cloudformation.json with the current test Richard is working on, in addition to usual code review.

**Comment by wallrj** [comment]
Thanks @sarum90 

I ran my automated test on the template from your branch and it passed.

```
CLOUDFORMATION_TEMPLATE_URL=https://s3-us-west-1.amazonaws.com/mewert-chq-work-bucket/FLOC-3913-cloudformation.json AWS_DEFAULT_REGION=us-west-1 KEY_PAIR=... ACCESS_KEY_ID=... SECRET_ACCESS_KEY=...  trial flocker.acceptance.endtoend.test_installer
flocker.acceptance.endtoend.test_installer
  DockerComposeTests
    test_docker_compose_up_postgres ... Warning: Permanently added '54.153.116.251' (ECDSA) to the list of known hosts.
                                   [OK]

-------------------------------------------------------------------------------
Ran 1 tests in 601.465s

PASSED (successes=1)
```

I'll merge this.

**Pull request merged**